rust/Cargo.toml.in

```diff
@@ -10,7 +10,10 @@ debug = true
 
 [features]
 lua = []
+external = ["ntp-parser"]
 
 [dependencies]
 nom = "~3.0"
 libc = "~0.2.0"
+
+ntp-parser = { version = "^0", optional = true }
```

Comment:
It reduces the downloaded crates if you don't use all the features.
Without the 'optional', cargo build will fetch and build the code for ntp-parser even if the feature is not active.